ChatSecure/Classes/View Controllers/OTRMessagesGroupViewController.m

```diff
@@ -44,6 +44,11 @@ - (void)setupGroupChat:(NSArray <NSString *>*)buddies account:(OTRAccount *)acco
     [self setThreadKey:self.threadKey collection:[OTRXMPPRoom collection]];
 }
 
+- (nullable OTRAccount *)accountWithTransaction:(nonnull YapDatabaseReadTransaction *)transaction {
```

Comment:
Ok, added a comment.

Hahaha, sorry, no, they ARE that old. I started looking at some group chat issues back then, but then the whole migration thingy came along and the changes never made it to a pull request...